src/gamekit.cs

```diff
@@ -855,13 +859,14 @@ interface GKSavedGame : NSCopying {
 	}
 
 	[NoWatch]
-	[NoTV]
 	[Protocol, Model]
 	[BaseType (typeof (NSObject))]
 	interface GKSavedGameListener {
+		[NoTV]
 		[Export ("player:didModifySavedGame:")]
 		void DidModifySavedGame (GKPlayer player, GKSavedGame savedGame);
 
+		[NoTV]
```

Comment:
Let me go back one step (I might have induced you into an error earlier)

This looks like the protocol does not really exists in tvOS and that Apple just annotated the members and not the type itself. In doubt let's open a radar and add a comment/link into the xtro data

Reply:
Yea this is confusing.

Also one comment in xtro said:

```
## all members are not available so the protocol is empty
## however this is confusing because some protocols have no members (so it can't just be ignored)
```

I was thinking they added the conformance (even if the protocol is empty on tvOS) for later.

But yea I can also revert that, open a radar and annotate xtro.

Reply:
it's tricky since exposing the protocol (if not supported) could be considered a private API (when apps are analyzed) on tvOS. Since I don't see any benefit from an empty protocol then let's play it safe :)